python_ta/__init__.py

```diff
@@ -272,21 +273,28 @@ def reset_linter(config=None, file_linted=None):
         ),
     )
 
+    absolute_checkers_path = Path("python_ta//checkers").absolute()
+    path_as_string = absolute_checkers_path.__str__()
     custom_checkers = [
-        "python_ta.checkers.forbidden_import_checker",
-        "python_ta.checkers.possibly_undefined_checker",
-        "python_ta.checkers.global_variables_checker",
-        "python_ta.checkers.forbidden_io_function_checker",
-        "python_ta.checkers.invalid_range_index_checker",
-        "python_ta.checkers.one_iteration_checker",
-        "python_ta.checkers.type_annotation_checker",
-        "python_ta.checkers.unnecessary_indexing_checker",
-        "python_ta.checkers.shadowing_in_comprehension_checker",
-        "python_ta.checkers.redundant_assignment_checker",
-        "python_ta.checkers.invalid_for_target_checker",
-        "python_ta.checkers.missing_space_in_doctest_checker",
-        "python_ta.checkers.pycodestyle_checker",
+        ("python_ta.checkers." + f[:-3])
```

Comment:
You can make the `f[:-3]` a bit more readable by using [os.path.splitext](https://docs.python.org/3/library/os.path.html#os.path.splitext).